modules/bim/lib/open_project/bim/bcf_xml/issue_reader.rb

```diff
@@ -45,7 +45,7 @@ def synchronize_with_work_package
       # If there are already errors during the BCF issue creation, don't create or update the WP.
       return if issue.errors.any?
 
-      self.is_update = issue.work_package.present?
+      self.is_update = issue.work_package.persisted?
```

Comment:
Note that this became necessary as setting a project on the issue creates a work package implicitly, as it's a `has_one :project through: :work_package` relation